src/mpid/ch4/shm/xpmem/xpmem_control.h

```diff
@@ -14,43 +14,23 @@
 #include "xpmem_pre.h"
 #include "xpmem_impl.h"
 #include "xpmem_recv.h"
+#include "xpmem_send.h"
 
-#undef FUNCNAME
-#define FUNCNAME MPIDI_XPMEM_ctrl_send_lmt_ack_cb
-#undef FCNAME
-#define FCNAME MPL_QUOTE(FUNCNAME)
-MPL_STATIC_INLINE_PREFIX int MPIDI_XPMEM_ctrl_send_lmt_ack_cb(MPIDI_SHM_ctrl_hdr_t * ctrl_hdr)
-{
-    int mpi_errno = MPI_SUCCESS;
-    MPIR_Request *sreq = (MPIR_Request *) ctrl_hdr->xpmem_slmt_ack.sreq_ptr;
-
-    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_XPMEM_CTRL_SEND_LMT_ACK_CB);
-    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_XPMEM_CTRL_SEND_LMT_ACK_CB);
-
-    XPMEM_PT2PT_DBG_PRINT("send_lmt_ack_cb: complete sreq %p\n", sreq);
-    MPID_Request_complete(sreq);
-
-  fn_exit:
-    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_XPMEM_CTRL_SEND_LMT_ACK_CB);
-    return mpi_errno;
-  fn_fail:
-    goto fn_exit;
-}
 
 #undef FUNCNAME
-#define FUNCNAME MPIDI_XPMEM_ctrl_send_lmt_req_cb
+#define FUNCNAME MPIDI_XPMEM_ctrl_do_send_lmt_req_cb
```

Comment:
None of these `FCNAME` / `FUNCNAME` macros are necessary anymore. They should all be removed from this PR (in the original XPMEM PR as well).

Reply:
I think `MPIR_Err_create_code` (when you use `MPIR_ERR_CHKANDJUMP` macro) may use this macro? Should we give up using `MPIR_ERR_CHKANDJUMP`?

Reply:
Those should all change to use `__FUNC__` instead. It does the same thing.

Reply:
You mean modify `MPIR_Err_create_code` to use `__FUNC__`? If so, do I need to change it in this PR?

Reply:
Don't modify the function. Modify the way they're called. That should be a separate patch in this PR for any of the old code and the same patch for any of the new code.